internal/fusefrontend_reverse/virtualfile.go

```diff
@@ -59,8 +73,10 @@ func (f *virtualFile) GetAttr(a *fuse.Attr) fuse.Status {
 	}
 	st.Ino = f.ino
 	st.Size = int64(len(f.content))
-	st.Mode = syscall.S_IFREG | 0400
+	st.Mode = virtualFileMode
 	st.Nlink = 1
 	a.FromStat(&st)
+	a.Owner.Uid = f.Uid
+	a.Owner.Gid = f.Gid
```

Comment:
That is the case for the gocryptfs.diriv files, but I need those fields for the gocryptfs.longname.*.name files. How should I do if not ?